.travis.yml

```diff
@@ -1,14 +1,11 @@
 language: go
 go:
   - 1.5.1
-branches:
-  only:
-    - master
```

Comment:
> Is it necessary to remove this?

#15 said "all PR builds" so I thought to remove that, but I'm remembering now that it means "all PRs against master" right? And if builder doesn't need it, then there ya go--I'll restore this section.